README.md

```diff
@@ -156,6 +156,21 @@ Then, navigate to the root directory of the project and run:
 pip install --editable '.[all]'
 ```
 
+###For Windows machine
```

Comment:
nit: Since this is conda installation instructions, lets add this to the conda section: https://github.com/Trusted-AI/AIF360#conda